server/routes/screenshot/html.py

```diff
@@ -33,13 +35,44 @@
 bp = flask.Blueprint("screenshot", __name__, url_prefix='/screenshot')
 
 
+def env_valid():
+  return os.environ.get('FLASK_ENV') in [
+      'autopush', 'local', 'test', 'webdriver'
+  ]
+
+
 @bp.route('/')
 def home():
   """List recent commits and diff url
 
   Optional url argument "base" to set a base commit sha for comparison.
   """
+  if not env_valid():
+    flask.abort(404)
   base_sha = request.args.get('base', '')
+  domain = request.args.get('domain', '')
+
+  if domain:
```

Comment:
So are we ignoring the base_sha if domain is set? Maybe add a comment about this?